### PR TITLE
Upgrade to Gradle 7.5.1

### PR DESCRIPTION
**Pull request by XN137**
See https://docs.gradle.org/7.5.1/release-notes.html

### CODE REVIEW

**Comment by ajantha-bhat** [comment]
@XN137 : Why are we manually doing it? I thought dependabot gonna raise a PR for this.

**Comment by XN137** [comment]
@ajantha-bhat afaik dependabot can update gradle plugins and project dependencies managed by gradle, but not gradle itself.

**Comment by ajantha-bhat** [comment]
> @ajantha-bhat afaik dependabot can update gradle plugins and project dependencies managed by gradle, but not gradle itself.

Got it. I saw the previous Gradle bumps PR now and figured it out. Thanks.

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
+# checksum was taken from https://gradle.org/release-checksums
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
-distributionSha256Sum=97a52d145762adc241bad7fd18289bf7f6801e08ece6badf80402fe2b9f250b1
+distributionSha256Sum=db9c8211ed63f61f60292c69e80d89196f9eb36665e369e7f00ac4cc841c2219
```
Comment (ajantha-bhat) [inline gradle/wrapper/gradle-wrapper.properties:4]: I guess this reorder is intentional to keep properties in sorted order.

**Review by ajantha-bhat — APPROVED** [review]
LGTM

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (XN137) [reply gradle/wrapper/gradle-wrapper.properties:4]: yes

**Pull request merged**